src/XIOPAK.CPP

```diff
@@ -1124,7 +1124,18 @@ SEC xfclear(	// Cleans the file by discarting all the data in the file
 	return xf->xflsterr;	/*    bad: report error */
 }
 //=============================================================================
-
+void xfcreatepath(			// Joins two directory path together
```

Comment:
The function name says one thing and the comment says another thing. Are you creating or joining? Pick one for consistency.